<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/icon/c_black.png</normaloff>:/new/icon/c_black.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Donate!</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="profile"/>
     </item>
     <item>
      <widget class="QComboBox" name="profilelist"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="jsonpath">
       <property name="text">
        <string>Json Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="jsonpathstring"/>
     </item>
     <item>
      <widget class="QToolButton" name="selectJsonFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>In file path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cryptolistread"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolcryptolistread">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Out file path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cryptolistwrite"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolcryptolistwrite">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Reports path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reportPath"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolReportPath">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Blacklist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="exchanges"/>
     </item>
     <item>
      <widget class="QTextEdit" name="blacklist"/>
     </item>
    </layout>
   </item>
   <item row="6" column="1" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Stake coin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="maincoins"/>
     </item>
     <item>
      <widget class="QTextEdit" name="maincoinslist"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="stake_coin">
         <property name="text">
          <string>BTC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="stake_coin_price">
         <property name="maximum">
          <number>10000000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="message">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>jsonpathstring</tabstop>
  <tabstop>selectJsonFile</tabstop>
  <tabstop>cryptolistread</tabstop>
  <tabstop>toolcryptolistread</tabstop>
  <tabstop>cryptolistwrite</tabstop>
  <tabstop>toolcryptolistwrite</tabstop>
  <tabstop>reportPath</tabstop>
  <tabstop>toolReportPath</tabstop>
  <tabstop>exchanges</tabstop>
  <tabstop>blacklist</tabstop>
  <tabstop>maincoins</tabstop>
  <tabstop>maincoinslist</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
